web/compose-compiler-integration/src/jsMain/kotlin/CrossmoduleTestsDependencies.kt

```diff
@@ -37,7 +37,7 @@ class ClassSavesComposableIntoVar(c: @Composable () -> Unit) {
 }
 
 class ClassSavesComposableIntoLateinitVar(c: @Composable () -> Unit) {
-    lateinit var composableVar: @Composable () -> Unit
```

Comment:
maybe it's better to suppress the warning here, because this a test and it needs to test lateinit var with Composable

Reply:
just for self-education - why it need to be lateinit? I mean, I am a simple man, I see tests passing, I'm joyous.

Reply:
it's a part of a test case.  Some time ago we had such vars broken and this test had been failing. 
Now it's fixed. So by keeping `lateinit` we ensure we catch it if it breaks again.